### ISSUE TITLE
gocritic: improve confusing error message

### ISSUE THREAD

**Issue by martinsirbe** [golangci/golangci-lint#950]
Thank you for creating the issue!

- [x] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [x] Yes, I've searched similar issues on GitHub and didn't find any.
- [x] Yes, I've included all information below (version, config, etc).

Please include the following information:

<details><summary>Version of golangci-lint</summary>

```console
$ golangci-lint --version
golangci-lint has version 1.23.1 built from 567904e on 2020-01-20T08:00:15Z
```

</details>

<details><summary>Config file</summary>

```console
$ cat .golangci.yml
linters-settings:
  gocritic:
    disabled-checks:
    - whyNoLint
```

</details>

<details><summary>Go environment</summary>
Not relevant...

</details>

<details><summary>Verbose output of running</summary>

```console
$ golangci-lint run -v
level=info msg="[config_reader] Used config file .golangci.yaml"
level=warning msg="Gocritic check \"whyNoLint\" was disabled by config, was it's not enabled, no need to disable it"
```

</details>

### Issue
I got the following error message:
```
Gocritic check \"whyNoLint\" was disabled by config, was it's not enabled, no need to disable it
```
Based on this: https://github.com/golangci/golangci-lint/blob/fd997edeeb4ac277823750d746394d80f03d7ec1/pkg/config/config_gocritic.go#L142

I'm not sure what exactly does "was it's not enabled" mean, can this be clarified?
Furthermore, could this be improved with steps which indicates how this error can be fixed, if this even counts as an error in the first place?

This should probably read something like:
```
Gocritic check \"whyNoLint\" was disabled by config, but this check is already disabled by default, therefore there is no need to explicitly disable it via config again.
```
_That's if I understand correctly what's the issue here._

**Comment by ernado** [comment]
Thank you! Your wording is much better, feel free to add a PR if you want.

**Comment by ernado** [comment]
Was resolved in #958
